examples/c/builtin_ordered_set_intersect.c

```diff
@@ -267,6 +235,25 @@ int main()
     char user1_output_id[BUFFER_SIZE] = {0};
     char serialized_response[BUFFER_SIZE] = {0};
     size_t serialized_response_len = BUFFER_SIZE;
+    const char *admin_user_id = "admin";
+    const char *admin_user_password = "teaclave";
+
+    /* Register */
+    ret = login(admin_user_id, admin_user_password, token, &token_len);
+    if (ret != 0) {
+        fprintf(stderr, "[-] Failed to login.\n");
```

Comment:
Such `fprintf(stderr, "[-]...` statement has been used everywhere so It may worth to wrap it up as a macro in `utils.h`.

Reply:
Good suggestion. I'll polish it in the following PRs.